lib/Mojo/WebSocketProxy/Backend/ConsumerGroups.pm

```diff
@@ -364,7 +365,9 @@ Subscription will be done only once within first request to backend server.
 
 sub wait_for_messages {
     my ($self) = @_;
-    $self->{already_waiting} //= $self->redis->subscribe([$self->whoami], $self->redis->curry::weak::on(message => $self->curry::weak::_on_message));
+    $self->{already_waiting} //=
+        $self->redis->subscribe(['myriad.' . $self->whoami, $self->whoami],
```

Comment:
Is it possible to only add the prefix based on `msg_group` so we subscribe to one channel?

Reply:
Yes, we can also pass the prefix and prepend to the `$self->whoami` in `_prepare_request_data`